lib/registration.pm

```diff
@@ -231,15 +231,15 @@ sub select_addons_in_textmode {
     if ($flag) {
         send_key_until_needlematch 'scc-module-area-selected', 'tab';
         send_key_until_needlematch "scc-module-$addon",        'down';
-        send_key 'spc';
+        wait_screen_change { send_key 'spc' };
```

Comment:
I think the issue is caused by wrong needle:
https://openqa.suse.de/tests/890007#step/scc_registration/6
The test actually didn't select LTSS addon